### PR TITLE
badgooooor/#40 Legistration process page -- overview section

### PR DESCRIPTION
**Pull request by badgooooor**
### Description
- From https://github.com/wevisdemo/parliament-watch/issues/40
- Add content in legistration process page -- overview section
- Note: I have a bit problem on exporting and optimize process images (the 3 steps one) that there is still some white background leaked to image. I've tried on both squoosh.app or ImageOptim and they didn't remove it 😢.

**Desktop**
image
image

**Mobile**
image
image

### CODE REVIEW

File: src/components/LegislativeProcess/ProcessCard.svelte [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	export let imgSrc: string;
+	export let title: string;
+</script>
+
+<div class="flex flex-col items-center bg-green-10 p-5">
+	<img src={imgSrc} alt={`process-step-${title}`} />
```
Comment (Th1nkK1D) [inline src/components/LegislativeProcess/ProcessCard.svelte:7]: In Svelte, you can do
```svelte
<img src={imgSrc} alt="process-step-{title}" />
```
which is more concise.

File: src/routes/legislative-process/+page.svelte [hunk 2]
```diff
@@ -36,14 +38,86 @@
...
 					<hr />
+					<p class="body-02 mt-6">
+						เว็บไซต์ Parliament Watch มุ่งนำเสนอกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา
+						เพื่อแสดงให้เห็นถึงกระบวนการทำงานด้านนิติบัญญัติในฐานะตัวแทนประชาชน
+						รวมถึงสะท้อนให้เห็นถึงความคิดเห็นของสมาชิกรัฐสภาต่อประเด็นต่างๆ ในทางกฎหมาย
+						ซึ่งเป็นเครื่องมือในการกำหนดทิศทางของสังคม ทั้งนี้
+						เว็บไซต์นี้จะไม่ปรากฎกฎหมายที่ถูกตราขึ้นผ่านคณะรัฐมนตรีและไม่ต้องผ่านการพิจารณาโดยรัฐสภา
+						เช่น พระราชกฤษฎีกา กฎกระทรวง และกฎหมายที่ตราโดยองค์กรปกครองส่วนท้องถิ่น
+					</p>
+					<img
```
Comment (Th1nkK1D) [inline src/routes/legislative-process/+page.svelte:117]: Need a margin between paragraph and image. Please refer to Figma.

image

File: src/routes/legislative-process/+page.svelte [hunk 3]
```diff
@@ -36,14 +38,86 @@
 						การออกกฎหมายตามรัฐธรรมนูญ 2560
 					</h2>
 					<hr />
+					<p class="body-02 mt-6">
```
Comment (Th1nkK1D) [inline src/routes/legislative-process/+page.svelte:41]: Some sentences in this paragraph, and also other paragraphs below, are bold. Please recheck with Figma.

**Review by Th1nkK1D — CHANGES_REQUESTED** [review]
Thanks for the PR! I left some comments. I also found an image issue. I will talk with the designer and get back to you since it is not correct when exporting from Figma. Image optimizer just compresses the image to have a smaller size so it won't help.

**Comment by badgooooor** [comment]
> Thanks for the PR! I left some comments. I also found an image issue. I will talk with the designer and get back to you since it is not correct when exporting from Figma. Image optimizer just compresses the image to have a smaller size so it won't help.

Please ping me if images are updated. I'll retry exporting it.

File: src/components/LegislativeProcess/ProcessCard.svelte [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	export let imgSrc: string;
+	export let title: string;
+</script>
+
+<div class="flex flex-col items-center bg-green-10 p-5">
+	<img src={imgSrc} alt="process-step-${title}" />
```
Comment (Th1nkK1D) [inline src/components/LegislativeProcess/ProcessCard.svelte:7]: Svelte doesn't use `$` in front of the variable bracket when used in an attribute. In this case, if the title = "ชื่อ", the alt will be "process-step-$ชื่อ" which I'm not sure if it is what you want.

Other things looking good now. I already reach out to the designer and they should fix the image problem. I'll let you know when it's ready so you can export the correct images.

**Comment by Th1nkK1D** [comment]
> Please ping me if images are updated. I'll retry exporting it.

Our designers fix it. Please try to export the images again.

**Comment by badgooooor** [comment]
Updated images.

image

image

**Review by Th1nkK1D — APPROVED** [review]

**Comment by Th1nkK1D** [comment]
LGTM! Thank you for another PR ;)

**Pull request merged**